modules/metrics/README.md

```diff
@@ -132,6 +137,7 @@ key | type | required | note
 --- | ---- | -------- | ----
 `metrics.server` | string list | optional | list of [`What`](src/main/java/com/spotify/apollo/metrics/semantic/What.java) names to enable; defaults to [ENDPOINT_REQUEST_RATE, ENDPOINT_REQUEST_DURATION, DROPPED_REQUEST_RATE, ERROR_RATIO]
 `metrics.precreate-codes` | int list | optional | list of status codes to precreate request-rate meters for, default empty
+`metrics.reservoir-ttl` | int | optional | When to purge old values from the histogram, default 300 seconds.
```

Comment:
The ttl effects the memory usage on the reservoir, this might be worth mentioning or put an upper bound on.

Reply:
👍 - I added a note about that.